adapters/InMobi/InMobiAdapter/GADMAdapterInMobiUtils.m

```diff
@@ -114,25 +125,32 @@ void GADMAdapterInMobiSetTargetingFromExtras(GADInMobiExtras *_Nullable extras)
 }
 
 void GADMAdapterInMobiSetTargetingFromConnector(id<GADMAdNetworkConnector> _Nonnull connector) {
-  if (connector.userGender == kGADGenderMale) {
-    [IMSdk setGender:kIMSDKGenderMale];
-  } else if (connector.userGender == kGADGenderFemale) {
-    [IMSdk setGender:kIMSDKGenderFemale];
-  }
-
-  if (connector.userBirthday != nil) {
-    NSDateComponents *components = [NSCalendar.currentCalendar
-        components:NSCalendarUnitDay | NSCalendarUnitMonth | NSCalendarUnitYear
-          fromDate:connector.userBirthday];
-    [IMSdk setYearOfBirth:components.year];
-  }
+    if (connector.userGender == kGADGenderMale) {
+        [IMSdk setGender:kIMSDKGenderMale];
+    } else if (connector.userGender == kGADGenderFemale) {
+        [IMSdk setGender:kIMSDKGenderFemale];
+    }
+    
+    if (connector.userBirthday != nil) {
+        NSDateComponents *components = [NSCalendar.currentCalendar
+                                        components:NSCalendarUnitDay | NSCalendarUnitMonth | NSCalendarUnitYear
+                                        fromDate:connector.userBirthday];
+        [IMSdk setYearOfBirth:components.year];
+    }
+    
+    GADMAdapterInMobiSetTargetingFromExtras([connector networkExtras]);
+    GADMAdapterInMobiSetIsAgeRestrcitedUser(connector.childDirectedTreatment);
+}
 
-  GADMAdapterInMobiSetTargetingFromExtras([connector networkExtras]);
+void GADMAdapterInMobiSetTargetingFromAdConfiguration( GADMediationAdConfiguration *_Nonnull adConfig) {
+    GADMAdapterInMobiSetTargetingFromExtras(adConfig.extras);
+    GADMAdapterInMobiSetIsAgeRestrcitedUser(adConfig.childDirectedTreatment);
 }
 
-void GADMAdapterInMobiSetTargetingFromAdConfiguration(
-    GADMediationAdConfiguration *_Nonnull adConfig) {
-  GADMAdapterInMobiSetTargetingFromExtras(adConfig.extras);
+void GADMAdapterInMobiSetIsAgeRestrcitedUser(NSNumber *_Nullable isRestricted) {
```

Comment:
Typo, also can rename this to `GADMAdapterInMobiSetIsAgeRestricted`.

Reply:
fixed